src/Resources/translations/SonataClassificationMediaBundle.de.xliff

```diff
@@ -1,11 +1,11 @@
-<?xml version="1.0"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file source-language="en" datatype="plaintext" original="" >
-        <body>
-            <trans-unit id="form.label_media">
-                <source>form.label_media</source>
-                <target>Media</target>
-            </trans-unit>
-        </body>
-    </file>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like your env misses the xml indent variable

Reply:
See https://github.com/sonata-project/SonataClassificationMediaBundle/blob/master/.travis.yml#L29

Reply:
Thanks 👍